### DISCUSSION TITLE
顛覆創意產業倫理創新--AI數位創作擁有版權?

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2050] [General]
。https://technews.tw/2024/05/09/adobe-firefly-image-3/
顛覆創意產業倫理創新，Adobe Firefly Image 3 能為數位創作的版權保護設新標竿？
。https://technews.tw/2024/04/07/can-ai-writing-gain-favor-from-readers/
AI 寫作能獲得讀者青睞？研究：人類寫的文章流量多出五倍

記得高三練習寫的學測作文時，有一篇題目就在說是AI生成圖畫是否有資格獲獎，當時AI才剛興起，得知這個消息時真的很震驚，因為只要告訴AI自己想要的主題、繪畫風格等，用幾道指令就能產出如此精湛的作品，還比人類創作所花的時間還短!

除了驚訝，更多的是認為似乎有些不公平。其他參賽者都是親手一筆一畫完成作品，反觀用AI生成的畫作，便會覺得有些偷吃步。但後來想想，AI創作的作品也是作者親自下指令、花時間完成的，也是有原創的部分在內，要說這幅AI生成畫作有版權也是可以接受的。
而且，後來陸續出現許多AI生圖的軟體，從Midjourney、Dall-E，到Leonardo.Ai、Image Creator、MyEdit等，甚至有不須下指令、只要畫幾筆草稿就生圖的Vivid AI，功能進化快速且方便使用，也有更多軟體支援中文指令，未來，便只會有更多以AI生成的數位媒體出現，所以我認為AI作品擁有版權也是必要的。
但是，若AI只是在生成圖片上加上一棵樹，是否也能擁有版權？這就衍生出了類似“合理使用範圍”的問題。
第二篇文章結尾提出的方法或許可行--為AI生成內容設立「次級版權」(secondary rights in copyright)，讓縮短AI作品的保護期和縮小權利範圍。AI科技法這領域還有一大段路要走呢!

**Comment by hunteryang0108** [comment]
版權的存在是在保護自然人或是法人的權利，AI目前並不享有這些權利，不過操作AI的人是否能合法享有這些權利仍需要更多的討論。設立次級版權是一個很不錯的想法，既不失保護創作者的初衷，又能確保其他人的權利，是個兩全其美的辦法。

**Comment by Jerryguo888** [comment]
AI生成圖畫引發了關於版權和公平性的討論，雖然AI能快速創作，但是否應該賦予其版權仍具爭議，而AI的創作過程由人類指令主導，其原創性和版權問題還需要進一步探討。另外設立次級版權來合理規範AI生成內容的保護範圍，或許是解決這一問題的一個可行方法。在未來，法律也需要跟上科技發展，平衡創作權益與技術應用。
